### ISSUE TITLE
truffle --version crashes, truffle help doesn't crash

### ISSUE THREAD

**Issue by agro1986** [trufflesuite/truffle#5186]
## Issue

`truffle --version` crashes

## Steps to Reproduce

run this:

```
truffle --version
```

## Actual Results

```
TypeError: Cannot read properties of null (reading 'name')
    at prepareOptions (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/truffle/build/webpack:/packages/core/lib/command-utils.js:86:39)
    at Object.586806 (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/truffle/build/webpack:/packages/core/cli.js:65:1)
    at __webpack_require__ (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/truffle/build/webpack:/webpack/bootstrap:19:1)
    at /Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/truffle/build/webpack:/webpack/bootstrap:37:1
    at Function.__webpack_require__.O (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/truffle/build/webpack:/webpack/runtime/chunk loaded:23:1)
    at __webpack_require__.x (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/truffle/build/webpack:/webpack/bootstrap:38:1)
    at Function.__webpack_require__.x (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/truffle/build/webpack:/webpack/runtime/startup chunk dependencies:36:1)
    at /Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/truffle/build/webpack:/webpack/startup:3:1
    at Object.<anonymous> (/Users/xxx/.nvm/versions/node/v16.15.1/lib/node_modules/truffle/build/cli.bundled.js:628:12)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
```

## Environment

* Operating System: MacOS 12.4 (21F79)
* Truffle version (`truffle version`): v5.5.17
* node version (`node --version`): v16.15.1
* npm version (`npm --version`): 8.11.0

**Comment by agro1986** [comment]
Hi @fainashalts . If I write `truffle help` it shows:

```
  --version  Show version number                                       [boolean]
```

so you might want to fix the docs also

**Comment by eggplantzzz** [comment]
Thanks @agro1986! Yes we bumped yargs recently and it added some extra stuff to the help display - some of which we haven't dealt with yet :(

**Comment by cliffoo** [comment]
Fixed in [pull#5266](https://github.com/trufflesuite/truffle/pull/5266) and in the [latest release v5.5.22](https://github.com/trufflesuite/truffle/releases/tag/v5.5.22). Thanks @agro1986 for opening this issue!
